RELEASE-NOTES.md

```diff
@@ -13,6 +13,18 @@
   * Added `create` command for `image-repository`.
   * Added `set (property)`, `unset (property)`, `suspend` and `resume` commands for `compute-pool`.
   * Added `upgrade` and `list-endpoints` commands for `service`.
+* Connections parameters are also supported by generic environment variables:
+  * `SNOWFLAKE_ACCOUNT`
+  * `SNOWFLAKE_USER`
+  * `SNOWFLAKE_PASSWORD`
+  * `SNOWFLAKE_DATABASE`
+  * `SNOWFLAKE_SCHEMA`
+  * `SNOWFLAKE_ROLE`
+  * `SNOWFLAKE_WAREHOUSE`
+  * `SNOWFLAKE_MFA_PASSCODE`
```

Comment:
why did you not add envs for `private key path` and `authenticator`?

Reply:
Good question, my thinking was that won't be necessary as there's no "security" issue with using those in command line

Reply:
No longer valid